tests/DependencyInstallerTest.php

```diff
@@ -164,11 +160,11 @@ public function testInstallPackage(
     /**
      * @return array
      */
-    public function packageProvider(): array
+    public static function packageProvider(): array
     {
         return [
-            ['psr/log', '@stable', true, false],
-            ['psr/log', '@stable', false, false]
+            ['psr/log', '@stable', true, false, true],
```

Comment:
Please introduce a test case where it actually matters that you provide `true/false` for the new `$allowOverrideVersion`. Your current test is just -- I expect the same thing to happen regardless whether I pass on true or false

Reply:
Done